.travis.yml

```diff
@@ -17,12 +17,10 @@ language: java
 sudo: false
 
 jdk:
-  - openjdk7
```

Comment:
Why did you removed the build for jdk7 and 8?
If lang3 still support them then it should be runned tests on 7 and 8

Reply:
With the proposed change, the build can only occur on Java 9. (Java 9 is needed to compile `module-info.java`).

Reply:
Sounds sad :( Especially because, I guess, jdk8 compiler willn't just ignore the `module-info.java`.
Maybe some other projects will even try to generate the `module-info.java`

Reply:
I think we should discuss this on the developer mailing list.